index.html

```diff
@@ -670,9 +684,10 @@ <h2>Profiles</h2>
         </p>
 
         <pre class="example nohighlight" title="Basic structure of a profile">
-{
-  "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-  "type": ["VerifiableProfile"],
+{"verifiableProfile": {
```

Comment:
Highly problematic, I get what you're trying to do here, but this is problematic from a data model perspective as well as how the rest of the world (10%-20% of the top 1 million domains now) are using JSON-LD.

Reply:
Maybe it is better to cater for the top 80-90% ??

Reply:
This is a false choice. We can have it both ways, there are many examples of it working in the past.